delivery_schenker/data/delivery_schenker_data.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- Schenker Product Packagings. It'd be nice to have proper measures -->
+    <record id="schenker_packaging_01" model="product.packaging">
+        <field name="shipper_package_code">CI</field>
+        <field name="name">Canister</field>
```

Comment:
As we can have other module adding similar packagings, put at least in the name between parenthesis the carrier name for distinguishing them.

Reply:
Done